clients/da-vinci-client/src/main/java/com/linkedin/davinci/kafka/consumer/ActiveActiveStoreIngestionTask.java

```diff
@@ -193,59 +153,9 @@ public static int getKeyLevelLockMaxPoolSizeBasedOnServerConfig(VeniceServerConf
         * serverConfig.getKafkaClusterIdToUrlMap().size() * multiplier + 1;
   }
 
-  @Override
-  protected DelegateConsumerRecordResult delegateConsumerRecord(
```

Comment:
The change is too big and is a bit confusing with this override method. I think you moved these all into StorePartitionDataReceiver class, but how do you do the override there then?